cogeo_mosaic/backends/file.py

```diff
@@ -1,33 +1,35 @@
-import functools
-import json
+"""cogeo-mosaic File backend."""
+
 from typing import Dict, Optional, Tuple, Union
 
+import json
+import functools
+
 import mercantile
 
+from cogeo_mosaic.model import MosaicJSON
 from cogeo_mosaic.backends.base import BaseBackend
 from cogeo_mosaic.backends.utils import (
     _compress_gz_json,
     _decompress_gz,
     get_assets_from_json,
 )
-from cogeo_mosaic.model import MosaicJSON
 
 
 class FileBackend(BaseBackend):
     """Local File Backend Adapter"""
 
-    def __init__(
-        self, path: str, mosaic_def: Optional[Union[MosaicJSON, Dict]] = None,
-    ):
+    def __init__(self, path: str, mosaic_def: Optional[Union[MosaicJSON, Dict]] = None):
+        """Initialize FileBackend."""
```

Comment:
Was this reformatted with black? It's surprising because I ran black on everything. Does black pick up the longer line-length here: https://github.com/developmentseed/cogeo-mosaic/blob/60749264bd8375c482ca5420ccefd39964edb3ce/tox.ini#L44
?

Reply:
yes. we usually use 90